life/modernized/03-testimonies.adoc

```diff
@@ -1,5 +1,8 @@
 == Testimonies Concerning Elizabeth Stirredge
 
+[.embedded-content-document.letter]
+--
```

Comment:
Because this is an entire chapter of testimonies, I don't think we need to embed them. I'm realizing that all this must be kinda confusing!! I'm going to slack you in a few and try to explain this concept better. But I think you can remove all of the embedded tags that you added to these testimonies at the end of her journal here.